src/Commands/HubspotCommerceCommands.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Drupal\commerce_hubspot\Commands;
+
+use Drupal\commerce_hubspot\Hubspot\ECommerceBridgeServiceInterface;
+use Drupal\commerce_hubspot\Hubspot\SyncToServiceInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Class HubspotCommerceCommands
+ * @package Drupal\commerce_hubspot\Commands
+ */
+class HubspotCommerceCommands extends DrushCommands {
```

Comment:
I think we should call this CommerceHubspot to continue with uniformity in the naming.

Reply:
`CommerceHubSpot` or `HubSpotCommerce` is not needed in this case; it's obvious, we're in the Commerce HubSpot module. It would be a lot more useful to name it something meaningful, what do the commands in this file do? We can add more commands in time and split them into multiple files. I'd propose `SyncCommands`.